js/app/features/bluesky/oauthResolver.tsx

```diff
@@ -0,0 +1,40 @@
+import type { OAuthAuthorizationServerMetadata } from "@atproto/oauth-client";
+import {
+  OAuthResolver,
+  ResolveOAuthOptions,
+} from "@atproto/oauth-client/dist/oauth-resolver";
```

Comment:
Guessing that the problem could be coming from is importing from `dist` here instead of the normal export? Regardless, happy to ship this if it works